Gemfile

```diff
@@ -32,6 +32,8 @@ gem 'pg', '~> 1.4'
 gem 'puma', '~> 6.0'
 gem 'rack-attack', '~> 6.6'
 gem 'react-rails', '~> 2.6'
+# Use Redis for Action Cable
+gem 'redis', '~> 5.0'
```

Comment:
Somewhat unrelated tidying, Dependabot put this on its own at the bottom of the gemfile